--- Makefile.orig	2011-05-17 06:00:34.000000000 +0300
+++ Makefile	2011-05-31 14:04:45.000000000 +0300
@@ -93,7 +93,7 @@
 
 install-aspell-dict: aspell
 	install -m 0755 -d  $(PREFIX)$(datadir) $(PREFIX)$(dictdir)
-	install -m 0644  $(ASPELL_DIST)/$(KOI8U_SET) $(ASPELL_DIST)/$(KOI8U_MAP) $(PREFIX)$(dictdir)
+	install -m 0644  $(ASPELL_DIST)/koi8-u-nl.cset $(ASPELL_DIST)/koi8-u-nl.cmap $(PREFIX)$(dictdir)
 	install -m 0644  $(ASPELL_DIST)/uk.dat $(PREFIX)$(datadir)
 	install -m 0644  $(ASPELL_DIST)/uk.rws $(ASPELL_DIST)/uk.multi $(ASPELL_DIST)/uk_affix.dat $(ASPELL_DIST)/ukrainian.alias $(ASPELL_DIST)/ukrainian.kbd $(PREFIX)$(dictdir)
 
